api/data_explorer/controllers/export_url_controller.py

```diff
@@ -113,41 +113,22 @@ def _get_entities_dict(cohort_name, query, filter_arr):
             # underscores here and periods in table_name attribute.
             'name': table_name.replace('.', '_').replace(':', '_'),
             'attributes': {
-                'table_name': table_name
+                'table_name': table_name,
+                'dataset_id': table_name.split('.')[1],
             }
         })
 
-    # If a cohort was selected, create a query entity and get Elasticsearch documents
-    # for the cohort so we can create sample_set entity.
+    # If a cohort was selected, create a query entity
     if cohort_name:
         entities.append({
             'entityType': 'cohort',
```

Comment:
Right now there's no SQL query for the all_participants cohort. Let's do:
- Rename _get_filter_query -> _get_sql_query
- In _get_sql_query, if filter is empty, pick any participant table and return participant_ids from it
- Add TODO like

```
TODO: _get_sql_query() assumes that all tables have all participant ids.
Make _get_sql_query() work if that's not the case. participant ids should be
aggregated from all tables.
```